docs/notebooks/sklearn_wrapper.ipynb

```diff
@@ -233,72 +239,163 @@
+   "metadata": {},
+   "source": [
+    "### Example of Using Pipeline"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 34,
    "metadata": {
     "collapsed": true
    },
    "outputs": [],
    "source": [
-    "def print_features(clf, vocab, n=10):\n",
+    "from sklearn.pipeline import Pipeline\n",
+    "from sklearn import linear_model\n",
+    "def print_features_pipe(clf, vocab, n=10):\n",
     "    ''' Better printing for sorted list '''\n",
-    "    coef = clf.coef_[0]\n",
+    "    coef = clf.named_steps['classifier'].coef_[0]\n",
+    "    print coef\n",
     "    print 'Positive features: %s' % (' '.join(['%s:%.2f' % (vocab[j], coef[j]) for j in np.argsort(coef)[::-1][:n] if coef[j] > 0]))\n",
-    "    print 'Negative features: %s' % (' '.join(['%s:%.2f' % (vocab[j], coef[j]) for j in np.argsort(coef)[:n] if coef[j] < 0]))"
+    "    print 'Negative features: %s' % (' '.join(['%s:%.2f' % (vocab[j], coef[j]) for j in np.argsort(coef)[:n] if coef[j] < 0]))\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "id2word=Dictionary(map(lambda x : x.split(),data.data))\n",
+    "corpus = [id2word.doc2bow(i.split()) for i in data.data]"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 38,
    "metadata": {
     "collapsed": false
    },
    "outputs": [
     {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING:gensim.models.ldamodel:too few updates, training might not converge; consider increasing the number of passes or iterations to improve accuracy\n"
+     ]
+    },
+    {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Positive features: clipper:1.50 code:1.24 key:1.04 encryption:0.95 chip:0.37 nsa:0.37 government:0.36 uk:0.36 org:0.23 cryptography:0.23\n",
-      "Negative features: baseball:-1.32 game:-0.71 year:-0.61 team:-0.38 edu:-0.27 games:-0.26 players:-0.23 ball:-0.17 season:-0.14 phillies:-0.11\n"
+      "[ -2.95020466e-01  -1.04115352e-01   5.19570267e-01   1.03817059e-01\n",
+      "   2.72881013e-02   1.35738501e-02   1.89246630e-13   1.89246630e-13\n",
+      "   1.89246630e-13   1.89246630e-13   1.89246630e-13   1.89246630e-13\n",
+      "   1.89246630e-13   1.89246630e-13   1.89246630e-13]\n",
+      "Positive features: Fame,:0.52 Keach:0.10 comp.org.eff.talk,:0.03 comp.org.eff.talk.:0.01 >Pat:0.00 dome.:0.00 internet...:0.00 trawling:0.00 hanging:0.00 red@redpoll.neoucom.edu:0.00\n",
+      "Negative features: Fame.:-0.30 considered,:-0.10\n",
+      "0.531040268456\n"
      ]
     }
    ],
    "source": [
-    "clf=linear_model.LogisticRegression(penalty='l1', C=0.1) #l1 penalty used\n",
-    "clf.fit(X,data.target)\n",
-    "print_features(clf,vocab)"
+    "model=SklearnWrapperLdaModel(num_topics=15,id2word=id2word,iterations=50, random_state=37)\n",
```

Comment:
PEP8: spaces around assignment operator ` = `. Other space/formatting/PEP8 issues further down this file, but this is the last comment.